### PR TITLE
Some more YUI removal

### PR DESCRIPTION
**Pull request by huwshimi**
Replaced methods for:
- Y.io
- Y.UA
- Y.getLocation
- Y.Promise
- Y.some
- Y.each
- Y.later
- Y.soon

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- test that you can long click on an app circle
- test that you can deploy changes (the ecs should work)

File: jujugui/static/gui/src/app/models/models.js [hunk 1]
```diff
@@ -2946,7 +2946,6 @@ YUI.add('juju-models', function(Y) {
     'juju-view-utils',
     'juju-charm-models',
     'juju-env-api',
-    'promise',
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/models/models.js]: Doesn't appear to be used.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7171/

**Comment by jaycee** [comment]
QA OK. Thanks @hu

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7174/

**Comment by huwshimi** [comment]
Thanks @jaycee and @hatched :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**